### ISSUE TITLE
Can't see report Titles

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-AMANDA-V7#66]
![image.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/4768a5ba-8516-48c3-8c5c-6aeb1e9c8d9e)

**Comment by Nadin-Nader** [comment]
@TracyLinder I dont think that the column width could get wider. If the dev side of this works like Knack then maybe. What i do (may not be enough for the user but...) is i just hover over it. There is a pop up that comes up showing the whole name. Will this not be enough for the users?
